python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -584,6 +584,13 @@ def forward(self, x: torch.Tensor):
                 # current workaround to reduce first token latency of fp32 input
                 # sometimes fp16 cause nan and training instability
                 # disable the conversion when training
+
+                # empty_cache on arc for models with large vocabulary (baichuan/qwen)
+                # for long input sequences.
+                do_empty_cache = self.weight._shape[0]*self.weight._shape[1] >= 70000*4096 \
```

Comment:
May be make `get_xpu_device_type(x0) == "arc" and self.weight._shape[0]*self.weight._shape[1] >= 70000*4096` as an attribute of LowBitLinear?

Reply:
Updated, take a look?